src/coreclr/jit/emitxarch.cpp

```diff
@@ -1917,53 +1766,37 @@ unsigned emitter::emitGetRexPrefixSize(instruction ins)
 }
 
 //------------------------------------------------------------------------
-// emitGetEvexPrefixSize: Gets Size of Evex prefix in bytes
-// TODO-XArch-AVX512: Once evex encoding is supported fully, this will take the place of emitGetAdjustedSize()
+// emitGetEvexPrefixSize: Gets Size of EVEX prefix in bytes
 //
 // Arguments:
-//    ins   -- The instruction
-//
-// Returns:
-//    Prefix size in bytes.
+//    ins   -- The instruction descriptor
 //
-unsigned emitter::emitGetEvexPrefixSize(instruction ins)
-{
-    if (IsEvexEncodedInstruction(ins))
-    {
-        return 4;
-    }
-
-    // If not AVX512, then we don't need to encode prefix.
-    return 0;
-}
-
-// Size of vex prefix in bytes
-unsigned emitter::emitGetVexPrefixSize(instruction ins, emitAttr attr)
-{
-    if (IsVexEncodedInstruction(ins))
-    {
-        return 3;
-    }
-
-    // If not AVX, then we don't need to encode vex prefix.
-    return 0;
+// Returns:
+//    Prefix size in bytes.
+//
+unsigned emitter::emitGetEvexPrefixSize(instrDesc* id)
+{
+    instruction ins = id->idIns();
+    assert(IsEvexEncodedInstruction(ins));
+    return 4;
```

Comment:
We only called this from `emitGetAdjustedSize` and only under an existing `IsEvexEncodedInstruction` check, so I simplified it to just assert and return the constant.